core/templates/core/includes/navbar.html

```diff
@@ -23,6 +23,9 @@
                 <li class="nav-item {% if current == 'Market' %}active{% endif %}">
                     <a class="nav-link" href="/market">Market</a>
                 </li>
+                <li class="nav-item {% if current == 'Balance' %}active{% endif %}">
+                    <a class="nav-link" href="/balance">Balance</a>
```

Comment:
I think it's better to name this feature (and hence, the tab) as "Transactions". Please change balance to transactions at all places.